File: Acceso-a-Datos-master/EVA2/Colegio_API/Colegio_API/Models/Alumno.cs
using System.Text.Json.Serialization;

namespace Colegio_API.Models {
    public class Alumno {
        public int ID { get; set; }
        public string Name { get; set; }
        public string Street { get; set; }
        public DateTime Date { get; set; }
        public string Gender { get; set; }
        [JsonIgnore]
        public Curso Curso { get; set; }
    }
}

File: angular/src/classes/alumno.ts
import { IAlumno } from "../interfaces/IAlumno";
import { Curso } from "./curso";


export class Alumno implements IAlumno {

  id: number
  name: string;
  street: string;
  date: string;
  gender: string;
  curso: Curso;

  constructor(id: number, name: string, street: string, date: string, gender: string, curso: Curso) {

    this.id = id;
    this.name = name;
    this.street = street;
    this.date = date;
    this.gender = gender;
    this.curso = curso
  }

  edad(): number {
    return 0;
  }
}
